app/src/main/java/de/danoeh/antennapod/adapter/DownloadStatisticsListAdapter.java

```diff
@@ -0,0 +1,88 @@
+package de.danoeh.antennapod.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.glide.ApGlideSettings;
+import de.danoeh.antennapod.core.storage.DBReader;
+import de.danoeh.antennapod.core.util.Converter;
+
+/**
+ * Adapter for the download statistics list
+ */
+public class DownloadStatisticsListAdapter extends StatisticsListAdapter {
+
+    public DownloadStatisticsListAdapter(Context context) {
+        super(context);
+    }
+
+    @Override
+    public int getItemCount() {
+        return statisticsData.feedTime.size() + 1;
+    }
+
+    public DBReader.StatisticsItem getItem(int position) {
+        if (position == 0) {
+            return null;
+        }
+        return statisticsData.feedTime.get(position - 1);
+    }
+
+    @NonNull
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(context);
+        if (viewType == TYPE_HEADER) {
+            View view = inflater.inflate(R.layout.statistics_listitem_total, parent, false);
+            TextView totalText = view.findViewById(R.id.total_description);
+            totalText.setText(R.string.total_size_downloaded_podcasts);
+            return new HeaderHolder(view);
+        }
+        return new StatisticsHolder(inflater.inflate(R.layout.statistics_listitem, parent, false));
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull RecyclerView.ViewHolder h, int position) {
```

Comment:
This method is copied to both child classes. Therefore, adding the colorful dots will add more code duplication. Maybe this can be extracted to some template methods, too.